mailer.http.class.php

```diff
@@ -218,14 +217,32 @@ protected function handle_response($response)
     protected function get_recipients()
     {
         $recipients = array();
+        $recipients_header_to = array();
 
         foreach ($this->to as $to) {
-            $recipients[] = array(
-                'address' => array(
-                    'email' => $to[0],
-                    'name' => $to[1]
-                ));
+            $recipients[] = $this->build_recipient($to[0], $to[1]);
+
+            // prepare for header_to
+            if(!empty($to[1])) {
+              $recipients_header_to[] = sprintf('%s <%s>', $to[1], $to[0]);
+            } else {
+              $recipients_header_to[] = $to[0];
+            }
+        }
+
+        // include bcc to recipients
+        // sparkposts recipients list acts as bcc by default
+        $bcc = $this->get_bcc(implode(',', $recipients_header_to));
+        if(!empty($bcc)) {
+          $recipients = array_merge($recipients, $bcc);
+        }
+
+        // include cc to recipients
+        $cc = $this->get_cc();
```

Comment:
we need to have `header_to` in cc too. https://github.com/SparkPost/php-sparkpost/blob/master/lib/SparkPost/Transmission.php
https://support.sparkpost.com/customer/en/portal/articles/2432290-using-cc-and-bcc-with-the-rest-api?_ga=1.2900533.446344954.1464029460